### ISSUE TITLE
παιδευόμενον only part. sg

### ISSUE THREAD

**Issue by lvcivs** [lvcivs/verb-o-matic#4]
I'm not sure but shouldn't παιδευόμενον be part. pres. med-pass. nom. sg. n. AND part. pres. med-pass. acc. sg. n. because neutral words share the same ending for nom and acc?

[moved from Google Code]

**Comment by lvcivs** [comment]
Thank you for reporting this. Yes, you are right. Actually, there are even more possibilties: besides nom. sg. n., it could be acc. sg. n., acc. sg. m. or voc. sg. n.

When I designed the exercise, I intended to ask only for the nominative form of the participles. But I see now that it leads to a problem when someone gives an additional solution, which the software will then classify as wrong, although it is correct.

I think the easiest way to fix this is to simply remove the user's option of choosing dat. or acc., as they are not used in the existing exercises at all (only nom. and gen. are used; voc. has never been there to choose). That way, the users cannot add one of the 'alternate' solutions and the software therefore cannot label them as wrong anymore.

The alternative would be to add all the alternative solutions to the exercises, but that would make them mandatory, which would make it much harder to get a 'correct' answer. I therefore prefer the other solution.
